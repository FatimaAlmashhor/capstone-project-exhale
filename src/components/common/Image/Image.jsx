import React from 'react' ;
import PropTypes from 'prop-types' ;

export default function Image({children , width}) {
    return (
        <div className = {`w-${width} h-auto `}>
            {children}
        </div>
    )
}

Image.propTypes = {
    children : PropTypes.element.isRequired ,
    width : PropTypes.string.isRequired
}