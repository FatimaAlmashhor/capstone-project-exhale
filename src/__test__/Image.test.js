import React from 'react';
import renderer from 'react-test-renderer';
import Image from '../components/common/Image';
it('renders correctly', () => {
  const Ex = <div>Example</div>
  const tree = renderer
    .create(<Image width = {' '}>{Ex}</Image>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});