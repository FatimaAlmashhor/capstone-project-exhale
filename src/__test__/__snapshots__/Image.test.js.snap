// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="w-  h-auto "
>
  <div>
    Example
  </div>
</div>
`;
